data/mods/Dark-Skies-Above/overrides/items/carnivore.json

```
@@ -18,7 +18,7 @@
   {
     "type": "COMESTIBLE",
     "id": "dry_meat_tainted",
-    "name": "dehydrated alien meat",
+    "name": { "str": "dehydrated alien meat" },
```

```suggestion
    "name": { "str_sp": "dehydrated alien meat" },
```
The same for other meats/fats/tallows/lards